docs/contracts/inheritance.rst

```diff
@@ -6,13 +6,19 @@ Inheritance
 
 Solidity supports multiple inheritance including polymorphism.
 
-All function calls are virtual, which means that the most derived function
-is called, except when the contract name is explicitly given or the
-``super`` keyword is used.
-
-All functions overriding a base function must specify the ``override`` keyword.
+Polymorphism means that a function call (internal and external)
+always executes the function of the same name (and parameter types)
+in the most derived contract in the inheritance hierarchy.
+This has to be explicitly enabled on each function in the
+hierarchy using the ``virtual`` and ``override`` keywords.
 See :ref:`Function Overriding <function-overriding>` for more details.
 
+It is possible to call functions further up in the inheritance
+hierarchy internally by explicitly specifying the contract
+using ``ContractName.functionName()`` or using ``super.functionName()``
```

Comment:
Should we mention that one may need to change them from ``external`` to ``public`` for this here or in the "How to update your code" part? Or we can add the ``= Base.superFunction;`` later, once we get around doing that...

Reply:
"possible to call ... internally ..." - that's of course not possible if the function cannot be called internally at all. This sentence here is unrelated to any 0.6.0 change.